scripts/content-auditor/content-auditor.py

```diff
@@ -0,0 +1,63 @@
+import os
+import requests
+import json
+import sys
+
+BASE_DIRECTORY = os.path.abspath(os.path.join(os.getcwd(), os.pardir, os.pardir))
+QUESTION_FILE = "content-auditor-prompt.txt"
+OUTPUT_FILE = "auditor-responses.md"
+FOUND_FILES = "auditor-found-files.txt"
```

Comment:
Maybe add a gitignore for these files, we dont want them pushed.

What do you think?

Reply:
Great plan. Done.